Artifacts/semver.artifactbundle/info.json

```diff
@@ -0,0 +1,19 @@
+{
+    "schemaVersion": "1.0",
+    "artifacts": {
+        "semver": {
+            "version": "3.4.0",
+            "type": "executable",
+            "variants": [
+                {
+                    "path": "semver-3.4.0-linux-gnu/bin/semver.sh",
+                    "supportedTriples": ["x86_64-unknown-linux-gnu"]
```

Comment:
I'm not totally sure of this tuple, but it works for the default GitHub ubuntu runner